models/persistentvolume.js

```diff
@@ -2,47 +2,125 @@ import { PVC } from '@/config/types';
 
 export const VOLUME_PLUGINS = [

```

Comment:
Sorry if I am a bit late to the game on the requirements here but do you think it would be beneficial to distinguish supported/unsupported in the dropdown labels themselves? I think this would be useful for users to know what is what for both PV's and StorageClasses

Reply:
The requirements are just to port. But It's probably a good suggestion.

Reply:
![image](https://user-images.githubusercontent.com/55104481/111233542-69268a00-85aa-11eb-99b9-6b883f2caf96.png)

Reply:
This also shows up in the list view but I think that's okay.

![image](https://user-images.githubusercontent.com/55104481/111234025-eeaa3a00-85aa-11eb-9273-f71ffb4c419e.png)